ckanext/nextgeoss/public/communityfeedback.js

```diff
@@ -118,7 +118,12 @@ function communityFeedback(catalogueID, catalogueNamespace, title)
     };
 
     if (title.length == 0) {
-      title = "Untitled";
+      try {
```

Comment:
While reviewing the code I noticed that the old behavior parsing the title of the individual feedback items is redundant, since the title has already been parsed and displayed when the feedback feed is processed, so we can just remove that part of `communityFeedback.getFeedbackItem` entirely.

I suggest removing lines 113 through 128 and line 230. I can do that in a separate PR, but maybe you'd rather do it as part of this one?